### DISCUSSION TITLE
Ducking when i speak

### DISCUSSION THREAD

**Discussion by quotequack** [wwmm/easyeffects#3595] [Q&A]
I want a way for all the selected outputs to lower their volume when i speak so i can then route that aurio to my microphone and for people to be able to hear me with lets say music in the background

**Comment by wwmm** [comment]
It seems to me that you want to do the same as #974. Put a compressor in the output effects pipeline and instead of using the internal source for the sidechain set your microphone as input for it. This way the compressor will act based on your voice. You will just have to adjust the amount of compression to your needs.
